docs/source/AdministratorGuide/Tutorials/installDFC.rst

```diff
@@ -0,0 +1,128 @@
+=================================
+Installing the DIRAC File Catalog
+=================================
+
+Pre-requisite
+=============
+
+You should:
+
+ * have a machine setup as described in :ref:`tuto_basic_setup`
+ * be able to install dirac components
+ * have installed a DIRAC SE using the tutorial (:ref:`tuto_install_dirac_se`).
+
+Tutorial goal
+=============
+
+The aim of the tutorial is to install the DIRAC FileCatalog (DFC)
+By the end of the tutorial, you will be able to do all sort of simple Data Management operations.
+
+More links
+==========
+
+More information can be found at the following places:
+
+ * Introduction to DataManagement: :ref:`data-management-system`
```

Comment:
```suggestion
* Introduction to DataManagement: :ref:`data-management-system`
```

Reply:
Don't you get now twice the "same" text, the reference link will have the text from the section title it refers to

Reply:
Yes, I will remove the text which is duplicated